src/plugins/remove/RemoveAction.js

```diff
@@ -87,6 +88,14 @@ export default class RemoveAction {
         this.openmct.objects.mutate(parent, 'composition', composition);
     }
 
+    removeFromCompositionAndSetSelection(parent, child) {
```

Comment:
If the remove action itself doesn't use this function, it probably shouldn't be on the RemoveAction class. It's behavior that's only needed from flex layouts right now, so it should probably be there.